types/options.ts

```diff
@@ -105,7 +105,10 @@ export interface EventHandlers {
     nickname: string,
     oldNickname?: string,
   ) => unknown;
-  presenceUpdate?: (data: PresenceUpdatePayload) => unknown;
+  presenceUpdate?: (
+    oldPresence: PresenceUpdatePayload | null,
```

Comment:
```suggestion
    presence: PresenceUpdatePayload,
```

Based on my suggestion above to be consistant with the rest of the lib its best to swap these around please.